src/server.c

```diff
@@ -5776,6 +5789,8 @@ sds genValkeyInfoString(dict *section_dict, int all_sections, int everything) {
                 "maxmemory:%lld\r\n", server.maxmemory,
                 "maxmemory_human:%s\r\n", maxmemory_hmem,
                 "maxmemory_policy:%s\r\n", evict_policy,
+                "key_eviction_memory:%lld\r\n", server.key_eviction_memory,
```

Comment:
now the config is `maxmemory-reserved`, here we should use `maxmemory_reserved` and `maxmemory_reserved_human` to avoid misunderstandings I think